internal/handlers/helpers.go

```diff
@@ -0,0 +1,11 @@
+package handlers
+
+// ArrayContainsString is a helper to find out if a given string is inside an array or not.
+func ArrayContainsString(list []string, key string) bool {
+	for _, value := range list {
+		if key == value {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
why is this in the handler package? this is a helper (although i hate this thing)